src/convertToObject.js

```diff
@@ -12,7 +12,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cssProperties = {};
+  const itemString = sourceString.split(';');
+  const keyAndValues = itemString.map(keyValue => keyValue.split(':'));
+  const keyAndValuesDefinite = keyAndValues.filter(keyValue =>
+    keyValue[0] !== undefined && keyValue[1] !== undefined);
```

Comment:
you can use chaining

```suggestion
  const itemString = sourceString
    .split(';')
    .map(keyValue => keyValue.split(':'))
    .filter(keyValue => keyValue[0] !== undefined && keyValue[1] !== undefined);
```